src/components/dialogs/generator/modification/generator-modification-dialog.js

```diff
@@ -308,10 +312,11 @@ const GeneratorModificationDialog = ({
         (equipmentId) => {
             if (equipmentId) {
                 setDataFetchStatus(FetchStatus.RUNNING);
-                fetchEquipmentInfos(
+                fetchNetworkElementInfos(
                     studyUuid,
                     currentNodeUuid,
-                    'generators',
+                    EQUIPMENT_TYPES.GENERATOR.type,
```

Comment:
@SlimaneAmar if we rename "equipment" into "network element", we should rename it everywhere, including the constants no ?